packages/pharos/src/components/toggle-button-group/pharos-toggle-button.ts

```diff
@@ -108,7 +99,6 @@ export class PharosToggleButton extends PharosButton {
         id="button-element"
         ?autofocus=${this.autofocus}
         ?selected=${this.selected}
```

Comment:
I don't believe native HTML buttons have a `selected` attribute. You can probably delete the whole `render` function since you are inheriting the one from `PharosButton`

Reply:
yeah, I was thinking I needed to set the type explicitly to `button` but it's already set in the constructor so that should be fine.